src/components/budgetTracker/BudgetTracker1L.java

```diff
@@ -0,0 +1,171 @@
+package components.budgetTracker;
+
+import java.util.Iterator;
+
+import components.stack.Stack;
+import components.stack.Stack2;
+
+/**
+ * {@code budgetTracker} represented as a {@link components.stack.Stack2
+ * components.stack.Stack2} with implementations of primary methods.
+ *
+ * @convention <pre>
+ * [$this.rep is not null]
+ * [$this.accountName is not null]
+ * </pre>
+ * @correspondence <pre>
+ * this = [value of $this.rep based on Stack]
+ * </pre>
+ */
+public class BudgetTracker1L extends BudgetTrackerSecondary {
+    /*
+     * Private members
+     */
+
+    /**
+     * The stack that represents the budget tracker.
+     */
+    private Stack<Transaction> rep;
+
+    /**
+     * Creator of initial representation.
+     */
+    private void createNewRep() {
+        this.rep = new Stack2<>();
+    }
+
+    /**
+     * The name of the account associated with this budget tracker.
+     */
+    private String accountName = "Default Account";
```

Comment:
I think it's probably okay to define your fields inline, but we typically do it in `createNewRep`.